### PR TITLE
Add Workarounds To Readme.md

### PR DESCRIPTION
**Pull request by Serkali-sudo**
I combined them all in the readme file.
https://github.com/PierfrancescoSoffritti/android-youtube-player/issues/885#issuecomment-1279762405

### CODE REVIEW

**Comment by PierfrancescoSoffritti** [comment]
Thanks for putting this together! can you send the PR to the dev branch? i will review it sometime this week :)

File: README.md [hunk 1]
```diff
@@ -941,6 +952,326 @@ In order to use your receiver you need a receiverId. This is the ID of your rece
...
+```js
+function hideCaption() {
+  if(!player) {
+    return;
+  }
+  player.loadModule('captions');
+}
+```
+
+## Play Next Video
```
Comment (PierfrancescoSoffritti) [inline README.md:1225]: it's not super clear to me what this workaround does, can you please try to explain a bit better? why would someone want to use it?

**Review by PierfrancescoSoffritti — COMMENTED** [review]
I have take the liberty to rephrase some some of the text. Let me know what you think.

The idea is to give general guidance instead of a step-by-step tutorial.

File: README.md (hunk 1 above)
Comment (Serkali-sudo) [reply README.md:1225]: that code basically gets YouTube video from "more video"(you can think of it like recommendation  section on YouTube) so if you want to play next recommended video.It is like "auto play" function on YouTube. I personally call that code from `YT.PlayerState.ENDED` so when video ends it automatically plays the next recommended video.

Comment (PierfrancescoSoffritti) [reply README.md:1225]: Ok that makes sense, thanks for the explanation.

**Comment by PierfrancescoSoffritti** [comment]
thanks for the help @Serkali-sudo

**Pull request merged**